ipaserver/install/plugins/update_managed_permissions.py

```diff
@@ -291,6 +291,14 @@
             'ipadomainlevel', 'objectclass',
         },
     },
+    'System: Grant add on CAS container': {
```

Comment:
IMO this should be kept together with other sub CA related permissions in `ipaserver/plugins/ca.py`